contributing.md

```diff
@@ -58,7 +56,7 @@ Examples of minor corrections include:
 
 The Working Group uses pull requests to track changes. To submit a change to the specification:
 
-**3.1.** Create a topic branch prefixed with the issue-id included in the name (e.g. 4-resource-name-dimension)
+**3.1.** Create a topic branch prefixed with the issue-id included in the name (e.g. 4-resource-name-dimension). Ensure that the group has agreed the creation of a PR based on a particular Issue.
```

Comment:
s/has agreed the creation of/has agreed to create/ - wording

Reply:
```suggestion
**3.1.** Create a topic branch prefixed with the issue-id included in the name (e.g. 4-resource-name-dimension). Ensure that the group has agreed to create a PR based on a particular Issue.
```